tests/x11/reboot_gnome.pm

```diff
@@ -24,7 +24,7 @@ sub run() {
     if (get_var("SHUTDOWN_NEEDS_AUTH")) {
         assert_screen 'reboot-auth';
         wait_still_screen;
-        type_password;
+        type_string $testapi::password, max_interval => 5;
```

Comment:
it does work, but I would prefer being able to count the password stars in video